packages/edit-site/src/components/global-styles/screen-revisions/revisions-buttons.js

```diff
@@ -163,6 +163,7 @@ function RevisionsButtons( {
 					>
 						<Button
 							className="edit-site-global-styles-screen-revisions__revision-button"
+							__experimentalIsFocusable
```

Comment:
This button can contain a lot of important information and should be perceivable even when disabled.